### DISCUSSION TITLE
View response of webhook/request flow operation

### DISCUSSION THREAD

**Discussion by MidasVE** [directus/directus#19077] [Feature Requests]
### Summary

I would like to see the response of the request in the activity logs of a Flow when using webhook/request url operation.

### Motivation

It would be great to see what response the webhook is getting back, so we can better monitor scheduled webhook calls.

### Detailed Design

![image](https://github.com/directus/directus/assets/26002295/db2e3e0e-e045-463f-9c93-fca31717a687)
Add a new tab underneath this that opens and shows the response.

### Must Have:
Show the response somewhere in the UI
### Could Have:
Formatting

### Drawbacks

I don't know if this would be complex or not

### Alternatives

I don't know

### Adoption Strategy

I don't think it would be breaki

**Comment by shipsides-sparknel** [comment]
if i understand you correctly, you can do this already.

in the webhook you've created, set the Response Body to Data of Last Operation (or specify an operation by using {{ ref_name }}). then when your flow is complete it will return the output of the chosen operation to the webhook requestor.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
